.travis.yml

```diff
@@ -34,6 +34,7 @@ addons:
         # - libwebpmux2
       - libfftw3-dev
       - libglib2.0-dev
+      - python-gobject
```

Comment:
I'm not sure this is necessary, is it? pyvips does not use gobject-introspection.

Reply:
I tried without this line and it seems to work OK.